redirect/routes.php

```diff
@@ -250,6 +250,10 @@ class Route {
             'description' => 'Modellbildung und Simulation',
             'target'      => 'https://www5.in.tum.de/wiki/index.php/Modeling_and_Simulation_-_Summer_18',
         ],
+        'pp' => [
+            'description' => 'Protein Prediction 1',
+            'target'      => 'https://www.rostlab.org/teaching/ss18/pp1cs',
+        ]
```

Comment:
Next guy/gurl will for sure forget the colon :man_facepalming:

Reply:
I hope so, then our CI can blame him! 😊